app/src/main/java/fr/free/nrw/commons/utils/StringSortingUtils.java

```diff
@@ -50,15 +50,23 @@ private static double calculateSimilarity(String str1, String str2) {
      * @return Number of characters the strings differ by
      */
     private static int levenshteinDistance(String str1, String str2) {
-        if (str1.equals(str2)) return 0;
-        if (str1.length() == 0) return str2.length();
-        if (str2.length() == 0) return str1.length();
+        if (str1.equals(str2)) {
```

Comment:
There appear to be a lot of formatting changes in this PR. In my opinion these changes do make the code more readable, however they aren't relevant to the issue being addressed. Could you please revert the formatting changes and only keep the code changes? Thanks!